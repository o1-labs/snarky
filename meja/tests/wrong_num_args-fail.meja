type t('a, 'b) = ('a, 'b);

type u = t(int);
