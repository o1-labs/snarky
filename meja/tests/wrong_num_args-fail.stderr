File "tests/wrong_num_args-fail.meja", line 3, characters 9-15:
Error: The type constructor t expects 2 argument(s)
       but is here applied to 1 argument(s).
